nightfall-client/README.md

```diff
@@ -19,79 +19,39 @@ nightfall-client requires a number of services to be present for it to work. The
 instructions explain how to run all of these up, in a similar 'developer' mode with local file
```

Comment:
This README is intended for outside people trying to use the SDK. So, it needs to be very clear. Wha'ts developer mode?